src/main/resources/static/_shared.yml

```diff
@@ -2883,6 +2884,12 @@ components:
           type: string
         isWithdrawn:
           type: boolean
+        domainEventNote:
```

Comment:
should this not just be called domainEvent? Can we link it to the actual domain event definition?